src/main/java/picard/illumina/LanePhasingMetricsCollector.java

```diff
@@ -60,8 +59,13 @@ public LanePhasingMetricsCollector(final Collection<Tile> laneTiles) {
 
         // Calculate the medians for the collected data
         for (final TileTemplateRead tileTemplateRead : phasingValues.keySet()) {
-            medianPhasingMap.put(tileTemplateRead, medianPercentage(phasingValues.get(tileTemplateRead)));
-            medianPrePhasingMap.put(tileTemplateRead, medianPercentage(prePhasingValues.get(tileTemplateRead)));
+            if (usePercentage) {
+                medianPhasingMap.put(tileTemplateRead, medianPercentage(phasingValues.get(tileTemplateRead)));
```

Comment:
It might be clearer to make a new method that accepts `usePercentage` e.g. `median(Collection<Float> phaseValues, boolean usePercentage)`. That would avoid some duplicate code here too.